### PR TITLE
add DD dataset and SAGPool hyperparameters

### PR DESCRIPTION
**Pull request by duchenzhuang**
I want to add SAGPool to cogdl, but I found it has achieved. So what I do is a supplement. According to https://github.com/inyeoplee77/SAGPool.
First, I add the DD dataset to cogdl. I modify the match.yml and cogdl/datasets/pyg. So you can load the DD dataset and use it to do some experiments.
Second, I improve the performance by modify the hyperparameters by adding a new file: sagpool-dd.py which is proper for the DD dataset. I think the learning_rate and weight decay should like https://github.com/inyeoplee77/SAGPool.

### CODE REVIEW

**Comment by cenyk1230** [comment]
@duchenzhuang Thank you for your contribution. But unit tests for the DD dataset should be also added.
